### PR TITLE
[STREAM] Update signing utility documentation

### PR DESCRIPTION
**Pull request by bkrebsbach**
Update Stream documentation to reference new token signing API.

### CODE REVIEW

File: products/stream/src/content/viewing-videos/securing-your-stream.md [hunk 1]
```diff
@@ -96,18 +99,56 @@ Replace the video ID with the signed token to use it.
...
+  },
+  "success": true,
+  "errors": [],
+  "messages": []
+}
 ```
 
-This endpoint accepts JSON bodies with the output from [Creating a signing key](#creating-a-signing-key) or any object with `pem` and `kid` attributes. To add a constraint, include it as a property of the body.
+The utility also takes a body, which accepts all the constraints defined [above](#signing-unique-tokens). If you've [created a signing key](#creating-a-signing-key), the `id` and `pem` fields from the response can be passed into the body to use a specific signing key. For more details about the allowed fields, see the [API docs](https://api.cloudflare.com/#stream-videos-create-a-signed-url-token-for-a-video).
```
Comment (marciocloudflare) [inline products/stream/src/content/viewing-videos/securing-your-stream.md:128]: to improve SEO, I would create the link as [which accepts all the constraints defined above]

File: products/stream/src/content/viewing-videos/securing-your-stream.md [hunk 2]
```diff
@@ -96,18 +99,56 @@ Replace the video ID with the signed token to use it.
...
+  },
+  "success": true,
+  "errors": [],
+  "messages": []
+}
+```
+
+Tokens generated with the signing utility have some constraints, which are listed below.
+
+- `exp` cannot be greater than 24 hours in the future from when the token is signed
```
Comment (marciocloudflare) [inline products/stream/src/content/viewing-videos/securing-your-stream.md:150]: Full sentences in lists should have period (full stop mark) at their end.

**Review by marciocloudflare — APPROVED** [review]
Approved with comments

**Comment by bkrebsbach** [comment]
@marciocloudflare Why was this merged? This needs to be reviewed by my team. Please revert this change ASAP.

**Comment by bkrebsbach** [comment]
@marciocloudflare I've reverted the change, please don't merge my PRs without checking with me first. My team didn't get a chance to look at this before it was merged.

**Comment by marciocloudflare** [comment]
@bkrebsbach  It was indeed merged, because I got a notification asking for approval from PCX. sorry about that

**Comment by marciocloudflare** [comment]
@bkrebsbach BTW, please check if you saw https://github.com/cloudflare/cloudflare-docs/pull/1434 or if I should go ahead and reverse the merger. Thank you

**Comment by bkrebsbach** [comment]
@marciocloudflare I think #1434 is fine to leave as-is.

**Pull request merged**